BAPeekPop/Classes/BAPreviewingViewController.m

```diff
@@ -92,7 +92,7 @@ - (void)viewDidLoad {
         
         // 2. add arrow-up icon
         self.arrowUpImageView = ({
-            UIImage *image = [UIImage imageNamed:@"icon-arrow-up" inBundle:[NSBundle bundleForClass:[BAPreviewingViewController class]] compatibleWithTraitCollection:nil];
+            UIImage *image = [UIImage imageNamed:@"icon_peek_pop_arrow_up"];
```

Comment:
Hi @BramYeh, I realized it's the same image as "icon-arrow-up", so I reverted the code back.